src/MSBuild.Conversion.Project/ProjectRootElementExtensionsForConversion.cs

```diff
@@ -11,8 +10,9 @@
 
 namespace MSBuild.Conversion.Project
 {
-    public static class ProjectRootElementExtensions
+    public static class ProjectRootElementExtensionsForConversion
     {
+
```

Comment:
just say no to blank lines

Reply:
![image](https://user-images.githubusercontent.com/6309070/81461968-42b2e980-9164-11ea-8ee6-5d6de5950c3a.jpeg)